k8s/namespaces/em/em-hrs-ingestor/prod-common.yaml

```diff
@@ -9,4 +9,4 @@ spec:
         IDAM_API_BASE_URI: "https://idam-api.platform.hmcts.net"
         OPEN_ID_API_BASE_URI: "https://hmcts-access.service.gov.uk/o"
         ENABLE_CRONJOB: true
-        MAX_FILES_TO_PROCESS: 2500
+        MAX_FILES_TO_PROCESS: 6000
```

Comment:
Should we just leave this at 2500 only? So that ccd-dat-store-api does not get overloaded while creating case?

Reply:
job will run only once, it will start at 11:30. we can not create that fast. 
and we have limit on our queue 100. after that we ignore the add requests.